internal/rbd/nodeserver.go

```diff
@@ -1396,9 +1399,46 @@ func getDeviceSize(ctx context.Context, devicePath string) (uint64, error) {
 	return size, nil
 }
 
-func (ns *NodeServer) SetReadAffinityMapOptions(crushLocationMap map[string]string) {
+func (ns *NodeServer) SetcmdReadAffinityMapOptions(crushLocationMap map[string]string) {
+	ns.cmdReadAffinityMapOptions = constructReadAffinityMapOption(crushLocationMap)
```

Comment:
Is there a good reason to keep `constructReadAffinityMapOption()` as a separate function? Why not make it

```go
func (ns *NodeServer) constructReadAffinityMapOption(crushLocationMap map[string]string)
```

or something similar?

Reply:
Certainly, we can make `cmdReadAffinityMapOptions` a public field. This would allow us to directly use it as follows:

```go
ns.cmdReadAffinityMapOptions = ns.constructReadAffinityMapOption(crushLocationMap)
```
This change would eliminate the need for the `SetcmdReadAffinityMapOptions()` method.